solidity/.prettierignore

```diff
@@ -8,3 +8,4 @@ hardhat-dependency-compiler/
 typechain/
 docgen-templates/
 export.json
+contracts/cross-chain/wormhole/L1BTCDepositorNttWithExecutor.sol
```

Comment:
Why are we ignoring this contract here?

Reply:
is an exception until we find why is still throwing a `[warn]` on the CI for prettier.